### PR TITLE
6 FatPipe CVE IDs

### PR DESCRIPTION
**Pull request by zmanion**
Updating CVE-2021-27860 and adding 5 other FatPipe CVE IDs. cc @sei-vsarvepalli.

### CODE REVIEW

File: 2021/27xxx/CVE-2021-27856.json [hunk 1]
```diff
@@ -1,18 +1,125 @@
 {
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27856.json:1]: 'problemtype' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/27xxx/CVE-2021-27856.json (hunk 1 above)
Comment (zmanion) [reply 2021/27xxx/CVE-2021-27856.json:1]: Fixed in https://github.com/CVEProject/cvelist/pull/3715/commits/7f6e785d430425b527782b8ec350c283baeca1f4

**Comment by cve-team** [comment]
WARNING: Github identified an issue when verifying the signature - unknown_key

**Comment by anthnysingleton** [comment]
Hey @zmanion, Can you take a look at this PR and resolve the Merge Conflicts. Once completed we can get this merged into the list. Thanks.

**Comment by zmanion** [comment]
> Hey @zmanion, Can you take a look at this PR and resolve the Merge Conflicts. Once completed we can get this merged into the list. Thanks.

Not sure how to do this, I don't have privileges to "Mark as resolved." The inbound 2021/27xxx/CVE-2021-27860.json from this PR should overwrite whatever is in master.

**Comment by ahouseholder** [comment]
> > Hey @zmanion, Can you take a look at this PR and resolve the Merge Conflicts. Once completed we can get this merged into the list. Thanks.
> 
> Not sure how to do this, I don't have privileges to "Mark as resolved." The inbound 2021/27xxx/[CVE-2021-27860](https://github.com/advisories/GHSA-hfgw-xgjr-v384).json from this PR should overwrite whatever is in master.

fixed merge conflict per @zmanion's  request

**Pull request merged**